mandatory/1-fix-functions.js

```diff
@@ -30,13 +30,15 @@ function greaterThan10(num) {
   }
```

Comment:
Hi Doris, I saw this comment and code in my email inbox just now and decided to comment since you said you were unsure, the function should return `true` or `false` based on whether the `num` is greater than 10 or not. `if / else` is one way to do it for sure but it could be simplified by just returning the condition.

```javascript
function greaterThan10(num) {
  return (num > 10);
}
```

This is because `(num > 10)` will equate to true or false, it's either truthy or falsy, we take a shortcut this way 😃